### PR TITLE
Added game - Shadow of Brimstone.

### PR DESCRIPTION
**Pull request by Kris248**
Submitting this PR means you have:

- [x] searched the [readme][readme-url] file and searched the [open pull requests][pr-url] (no duplicates please)
- [x] and will follow [the desired format][format-url]
- [x] used image links that resolve correctly, are appropriately sized, and are served over HTTPS
- [x] read [this year's Hacktoberfest welcome issue](https://github.com/edm00se/awesome-board-games/issues/269)
- [x] promised not to `@` me asking for review (I will see the notifications already, no need to add noise)

Once your PR is created, it will trigger a build a couple of automated checks, including a GitHub Action (Awesome-Lint). This includes a format linting process, so if there are any issues with the build, please address them. The others include a check on the submission limit and also a preview of the site with your changes; please check them out, they are tools tonhelp us both.

Do you have further questions? Check the [contribution guidelines][contributing-url] or [open an issue][new-issue-url].

When you're ready, check off the above acknowledgements and continue with your Pull Request... and thanks!

[pr-url]: https://github.com/edm00se/awesome-board-games/pulls
[readme-url]: https://github.com/edm00se/awesome-board-games/edit/main/readme.md
[format-url]: https://github.com/edm00se/awesome-board-games/blob/main/formatting.md
[contributing-url]: https://github.com/edm00se/awesome-board-games/blob/main/contributing.md
[new-issue-url]: https://github.com/edm00se/awesome-board-games/issues/new

### CODE REVIEW

File: .gitpod.yml
Comment (edm00se) [inline .gitpod.yml:1]: This file should not be here.

File: readme.md [hunk 1]
```diff
@@ -16,6 +16,25 @@
 
 ## Family
 
+### [Shadows of Brimstone: Swamps of Death (2014)](https://boardgamegeek.com/boardgame/15099shadows-brimstone-swamps-death)
```
Comment (edm00se) [inline readme.md:19]: The link is malformed. Also yremove the year from the end of the title.

File: readme.md [hunk 2]
```diff
@@ -16,6 +16,25 @@
...
+
+An exciting campaign system allows the players to visit local frontier towns between adventures, spending their hard-earned loot and building their characters from game to game! As players find fantastic gear and artifacts to equip their heroes, they also gain experience from their adventures. This experience is used to level up, guiding the hero's path through an expansive, class-specific upgrade tree of new skills and abilities, allowing each player to develop their hero to fit their own play style.
+
+In Swamps of Death, players will encounter portals to the otherworld Jargono, a humid swampland inhabited by sentient reptiles, vicious dinosaurs and a tribe of humanoids, cut off from return to their homeworld untold years after their own portal closed.
+
+So load up yer' six shooter, throw on yer' hat and poncho, and gather the posse as the darkness is coming, and all hell's about to break loose...in the Shadows of Brimstone!
+
+Can be used together with Shadows of Brimstone: City of the Ancients to raise the maximum players to 6.
```
Comment (edm00se) [inline readme.md:29]: The entirety of the description block needs to be commented, lines start with the `> ` character and a space.

**Review by edm00se — CHANGES_REQUESTED** [review]
There are a few things to fix, but most of that is formatting except for the extra file that should be removed.

**Comment by Kris248** [comment]
Can u review? Like this you want?

**Review by edm00se — APPROVED** [review]
Thanks for contributing!

refs #269

**Comment by Kris248** [comment]
i just added 1 game only. Can i add 10-20 games also like previous one?

**Pull request merged**